contracts/contracts/Job.sol

```diff
@@ -57,6 +61,12 @@ contract Job is IJob, Ownable {
         uint256 bounty;
         uint256 startTime;
         uint256 completedTime;
+        bool isReported;
```

Comment:
I think we should use another State here, instead of a flag.  

Currently, there is nothing that prevents the job from continuing.  It could still be paid out and completed while being reported.  We should block the job from continuing while in the reported state.

Reply:
Hmm, if the contract is paused, this means that if the report is declined, the supplier will have lost a couple of days until waiting for it to get resolved.
Would displaying a `Caution, job was reported` flag in the UI be enough ?